i6_models/parts/mha.py

```diff
@@ -0,0 +1,80 @@
+__all__ = ["MultiheadAttentionV1", "MultiheadAttentionV1Config"]
+
+import math
+
+import torch
+from torch import nn
+from i6_models.config import ModelConfiguration
+from i6_models.util import compat
+from dataclasses import dataclass
+
+
+@dataclass
+class MultiheadAttentionV1Config(ModelConfiguration):
+    input_dim: int
+    num_att_heads: int
+    att_weights_dropout: float
+    dropout: float
+
+    def __post_init__(self) -> None:
+        super().__post_init__()
+        assert self.input_dim % self.num_att_heads == 0, "input_dim must be divisible by num_att_heads"
+
+
+class MultiheadAttentionV1(nn.Module):
```

Comment:
Add some docstring why we need this and what it does.

Reply:
Added.